joomla/joomla-cms
# [5.1] Update deleted files list in script.php for 5.1.0-beta2 (2)

Pull Request for Issue # .

### Summary of Changes

Follow up PR to #42955 .

This pull request (PR) updates the list of files to be deleted on update in file `administrator/components/com_admin/script.php` to recent changes in the 5.1-dev branch in preparation for the upcoming 5.1.0-beta2 release.

In detail deleted files and folder from following PR's are added:

- From PR #42990 file
`/administrator/modules/mod_login/mod_login.php`
- From PR #43009 files
`/libraries/src/Event/Router/AfterInitialiseRouterEvent.php`
`/libraries/src/Event/Router/RouterEvent.php`
folder
`/libraries/src/Event/Router`
- From PR #42900 files
`/libraries/src/Http/HttpFactoryInterface.php`
`/libraries/src/Service/Provider/Http.php`
- From PR #42954 files
`/libraries/vendor/web-token/jwt-experimental/ContentEncryption/A128CCM_16_128.php`
`/libraries/vendor/web-token/jwt-experimental/ContentEncryption/A128CCM_16_64.php`
`/libraries/vendor/web-token/jwt-experimental/ContentEncryption/A128CCM_64_128.php`
`/libraries/vendor/web-token/jwt-experimental/ContentEncryption/A128CCM_64_64.php`
`/libraries/vendor/web-token/jwt-experimental/ContentEncryption/A256CCM_16_128.php`
`/libraries/vendor/web-token/jwt-experimental/ContentEncryption/A256CCM_16_64.php`
`/libraries/vendor/web-token/jwt-experimental/ContentEncryption/A256CCM_64_128.php`
`/libraries/vendor/web-token/jwt-experimental/ContentEncryption/A256CCM_64_64.php`
`/libraries/vendor/web-token/jwt-experimental/ContentEncryption/AESCCM.php`
`/libraries/vendor/web-token/jwt-experimental/KeyEncryption/A128CTR.php`
`/libraries/vendor/web-token/jwt-experimental/KeyEncryption/A192CTR.php`
`/libraries/vendor/web-token/jwt-experimental/KeyEncryption/A256CTR.php`
`/libraries/vendor/web-token/jwt-experimental/KeyEncryption/AESCTR.php`
`/libraries/vendor/web-token/jwt-experimental/KeyEncryption/Chacha20Poly1305.php`
`/libraries/vendor/web-token/jwt-experimental/KeyEncryption/RSAOAEP384.php`
`/libraries/vendor/web-token/jwt-experimental/KeyEncryption/RSAOAEP512.php`
`/libraries/vendor/web-token/jwt-experimental/LICENSE`
`/libraries/vendor/web-token/jwt-experimental/Signature/Blake2b.php`
`/libraries/vendor/web-token/jwt-experimental/Signature/ES256K.php`
`/libraries/vendor/web-token/jwt-experimental/Signature/HS1.php`
`/libraries/vendor/web-token/jwt-experimental/Signature/HS256_64.php`
`/libraries/vendor/web-token/jwt-experimental/Signature/RS1.php`
folders
`/libraries/vendor/web-token/jwt-experimental/Signature`
`/libraries/vendor/web-token/jwt-experimental/KeyEncryption`
`/libraries/vendor/web-token/jwt-experimental/ContentEncryption`
`/libraries/vendor/web-token/jwt-experimental`

### Testing Instructions

Code review.

Or if you want to make a real test, update a 5.1.0-beta1 or older version to the last 5.1 nightly build to get the actual result, and update a 5.1.0-beta1 or older version to to the update package built by Drone for this PR to get the expected result.

### Actual result BEFORE applying this Pull Request

The files and folders mentioned above are still present after updating from a 5.1.0-beta1 or older version.

### Expected result AFTER applying this Pull Request

The files and folders mentioned above have been deleted after updating from a 5.1.0-beta1 or older version.

### Link to documentations
Please select:
